README.md

```diff
@@ -195,13 +195,23 @@ wav         - speech files in wave file format
 
 ## Building for Windows on a Linux machine
 
-On Ubuntu 17 and above:
+On Ubuntu 20.04 LTS( 32bit ):
    ```
-   sudo apt-get install mingw-w64
+   sudo apt install mingw-w64 g++-mingw-w64
    mkdir build_windows && cd build_windows
-   cmake .. -DCMAKE_TOOLCHAIN_FILE=/home/david/freedv-dev/cmake/Toolchain-Ubuntu-mingw32.cmake -DUNITTEST=FALSE -DGENERATE_CODEBOOK=/home/david/codec2/build_linux/src/generate_codebook 
+   cmake -DCMAKE_TOOLCHAIN_FILE=~/Toolchain-Ubuntu-mingw32.cmake .. 
```

Comment:
Don't we need generate_codebook?

Reply:
Yes we should be generating codebooks.  Actually in https://github.com/drowe67/codec2/pull/285/commits/50528d97879f972e6da404fce43eaa79910478dc (part of #285) I have removed the static copies of these build time generated files.

Reply:
Also, I don't think it's necessary to have the toolchain file in ~. I was able to run `cmake -DCMAKE_TOOLCHAIN_FILE=../cmake/Toolchain-Ubuntu-mingw64.cmake ..` from the build folder without any problems.

But yeah, I was able to build without GENERATE_CODEBOOK, so no issue with removing it from this example.

Reply:
> Also, I don't think it's necessary to have the toolchain file in ~. I was able to run `cmake -DCMAKE_TOOLCHAIN_FILE=../cmake/Toolchain-Ubuntu-mingw64.cmake ..` from the build folder without any problems.
> 
> But yeah, I was able to build without GENERATE_CODEBOOK, so no issue with removing it from this example.

Thanks, @tmiw  I updated the readme!